conda/dali_python_bindings/recipe/meta.yaml

```diff
@@ -69,8 +69,8 @@ requirements:
     - patchelf
     - python-clang
     - git-lfs
-    - black =24.4.2
-    - black-jupyter =24.4.2
+    - black =24.10.0
```

Comment:
If you updating black - please https://github.com/NVIDIA/DALI/blob/main/docker/Dockerfile#L23 and https://github.com/NVIDIA/DALI/blob/main/.github/workflows/lint.yml#L22

Reply:
Done.